Carbon.FileSystem/Functions/Get-CNtfsPermission.ps1

```diff
@@ -0,0 +1,70 @@
+
+function Get-CNtfsPermission
+{
+    <#
+    .SYNOPSIS
+    Gets the permissions (access control rules) for a file or directory.
+
+    .DESCRIPTION
+    Permissions for a specific identity can also be returned. Access control entries are for a path's discretionary

```

Comment:
What happens if there are no permissions for that identity? An error? Nothing?

Reply:
Updated docs to explain. Currently, no error is written, but I should change that in Carbon.Permissions.